shared/types/VaccineDataTypes.ts

```diff
@@ -0,0 +1,17 @@
+export type DhbVaccineDoseData = {
+  dhbName: string;
+  firstDoses: number;
+  firstDosesPercentage: number;
+  firstDosesTo90Percent: number;
```

Comment:
Can we still have `hasMetFirstDosesTarget` etc?

Reply:
I wanted to keep the data as pure as possible since this is what will get saved into the txt file. 
I reckon we could had the `hasMetFirstDosesTarget` on a new type in the app which extends this.

We are event sourcing this haha the `hasMetFirstDosesTarget` is the current interpretation :P

Reply:
Yup that makes sense!

Reply:
Would be pretty nice to do in `getServerSideProps` etc